src/sponge/mod.rs

```diff
@@ -4,7 +4,10 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-mod sponge;
+pub mod sponge;
+pub mod truncated;
 
 pub use sponge::sponge_gadget as gadget;
 pub use sponge::sponge_hash as hash;
+pub use truncated::gadget as truncated_gadget;
```

Comment:
This is not needed. It makes more sense to call `truncated::hash` instead of the new alias. However, its optional and we should make single calls only after we make the breaking change of the alias removal